src/utils/Helper.ts

```diff
@@ -43,3 +44,15 @@ export function applySomeDefaults<T>(defaults: Partial<T>, partial?: Partial<T>)
   const complete = Object.assign(defaults, partial)
   return complete
 }
+
+export function mergeRequestOptions<T = CompleteRequestOptions | RequestOptions>(
+  aRequestOption: T,
```

Comment:
Could we call them target & source? Similar to how they are used in `Object.assign`. Because they will be used differently